sentry-sidekiq/lib/sentry/sidekiq-scheduler/scheduler.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+# Try to require sidekiq-scheduler to make sure it's loaded before the integration.
+begin
+  require "sidekiq-scheduler"
+rescue LoadError
+  return
+end
+
+# If we've loaded sidekiq-scheduler, but the API changed,
+# and the Scheduler class is not there, fail gracefully.
+return unless defined?(::SidekiqScheduler::Scheduler)
+
+module Sentry
+  module SidekiqScheduler
+    module Scheduler
+      def new_job(name, interval_type, config, schedule, options)
+        # Schedule the job upstream first
+        # SidekiqScheduler does not validate schedules
+        # It will fail with an error if the schedule in the config is invalid.
+        # If this errors out, let it fall through.
+        rufus_job = super
+
+        klass = config.fetch("class")
+        return rufus_job unless klass
+
+        # Constantize the job class, and fail gracefully if it could not be found
+        klass_const =
+          begin
+            Object.const_get(klass)
+          rescue NameError
+            return rufus_job
+          end
+
+        # For cron, every, or interval jobs — grab their schedule.
+        # Rufus::Scheduler::EveryJob stores it's frequency in seconds,
+        # so we convert it to minutes before passing in to the monitor.
+        monitor_config = case interval_type
+          when "cron"
+            Sentry::Cron::MonitorConfig.from_crontab(schedule)
+          when "every", "interval"
+            Sentry::Cron::MonitorConfig.from_interval(rufus_job.frequency.to_i / 60, :minute)
```

Comment:
ok only thing i'm not sure is if the server will accept for instance `300 minutes` and convert that to 5 hours or not? I asked, but we can patch that later

Reply:
I looked around, and in theory, we can have our own Fugit parser that would parse out the `schedule` string. But we'd have to get fugit as a dependency in. If we merge this, I would be down to test on my own and patch this up if it becomes an issue.

Reply:
yea we can do this later, no biggie

Reply:
300 minutes is fine, server will take care of it